src/SlashCommands.tsx

```diff
@@ -1026,6 +1028,27 @@ export const Commands = [
         },
         category: CommandCategories.actions,
     }),
+    new Command({
+        command: "confetti",
+        description: _td("Sends the given message with confetti"),
+        args: '<message>',
+        runFn: function(roomId, args) {
+            return success((async () => {
+                const cli = MatrixClientPeg.get();
+                const userId = cli.getUserId();
+                const userName = userId.slice(1).split(":").slice(0, 1);
```

Comment:
Seems strange to do it like this,
This seems like a good use for `m.emote`,

Try out `/me sends confetti`:
![image](https://user-images.githubusercontent.com/2403652/91156235-cebd3280-e6bb-11ea-8223-9334f765fde3.png)

That way the username is:
1. Not hardcoded into the event body
2. Not based on the MXID

Reply:
is it fine to be done like this: 
![Bildschirmfoto 2020-08-25 um 14 21 16](https://user-images.githubusercontent.com/34133773/91173571-5156e980-e6de-11ea-8071-9cbf40686d79.jpg) ?
basically the same as rainbowme

Reply:
i'm sorry if this will come out stupid, but /rainbow edits the same args. and it maps each char to be colored. could you please explain more how to do it for confetti?
does the function used in sendHtmlMessage should return a value for the confetti to work on the reciever side?

Reply:
Your latest screenshot looks fine, just use `sendEmoteMessage` instead as you're not specifying any HTML, just plain text it looks like.